core/platform/lf_zephyr_clock_counter.c

```diff
@@ -142,6 +142,8 @@ int _lf_clock_now(instant_t* t) {
     }
 
     *t = now_nsec;
+    clock_sync_apply_offset(t);
```

Comment:
Need to add to the function comment that the clock sync offset is added. Actually, this function comment seems incomplete overall.